package.json

```diff
@@ -14,9 +14,10 @@
     "pre-commit": "(pnpm run tsc || true) && (pnpm run lint || true) && pnpm run prettier"
```

Comment:
cc @ccatherinetan, maybe we want to remove this change & also the `package-lock.json` change? (do we want/need this change)?

Reply:
this comment is fully on the wrong line lol but i'm referring to the one on line 17/20 of this file

Reply:
yes sg! we'll delete the `@next/swc-darwin-arm64` line